rpm-testing/install.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+
+#
+# Determine if search capabilities are enabled
+#
+
+search_enabled="true"
+if [ "${SEARCH_ENABLED}" = "no" ] || [ "${SEARCH_ENABLED}" = "false" ] || [ "${SEARCH_ENABLED}" = "0" ]; then
+    search_enabled="false"
+fi
+echo "---------------- DEBUG: SEARCH"
+echo "Enabled: ${search_enabled}"
+echo "------------------------------"
+
+
+#
+# Configure repository
+#
+
+build_local_repository="false"
+if [ "${LOCAL_REPOSITORY}" = "yes" ] || [ "${LOCAL_REPOSITORY}" = "true" ] || [ "${LOCAL_REPOSITORY}" = "1" ]; then
+    build_local_repository="true"
+fi
+
+if [ "${build_local_repository}" == "true" ] ; then
+    sudo -u root bash -c 'cat << EOF > /etc/yum.repos.d/archivematica.repo
+[archivematica]
+name=archivematica
+baseurl=file:///am-packbuild/rpm/_yum_repository/
+enabled=1
+gpgcheck=0
+EOF'
+else
+    sudo -u root bash -c 'cat << EOF > /etc/yum.repos.d/archivematica.repo
+[archivematica]
+name=archivematica
+baseurl=https://packages.archivematica.org/1.7.x/centos
+gpgcheck=1
+gpgkey=https://packages.archivematica.org/1.7.x/key.asc
+enabled=1
+
+[archivematica-extras]
+name=archivematica-extras
+baseurl=https://packages.archivematica.org/1.7.x/centos-extras
+gpgcheck=1
+gpgkey=https://packages.archivematica.org/1.7.x/key.asc
+enabled=1
+EOF'
+fi
+
+sudo -u root yum update -y
+sudo -u root yum install -y epel-release policycoreutils-python
+
+
+#
+# SELinux tweaks
+#
+
+if getenforce; then
+    sudo semanage port -m -t http_port_t -p tcp 81
+    sudo semanage port -a -t http_port_t -p tcp 8001
+    sudo setsebool -P httpd_can_network_connect_db=1
+    sudo setsebool -P httpd_can_network_connect=1
+    sudo setsebool -P httpd_setrlimit 1
+fi
+
```

Comment:
We also should look if firewalld is running, and enable ports 81 and 8001 as shown in archivematica-docs:
```
sudo firewall-cmd --zone=public --add-port=81/tcp  --permanent
sudo firewall-cmd --zone=public --add-port=8001/tcp  --permanent
sudo service firewalld restart
```

Reply:
Done.

Reply:
It's at the bottom of the script FYI.